src/instance.rs

```diff
@@ -111,4 +110,20 @@ impl UserMeta {
             object,
         }
     }
+
+    /// Creates a new 'shell' of a user. The user does not exist as an object, and exists so that you have
+    /// a UserMeta object to make Rate Limited requests with. This is useful in scenarios like
+    /// registering or logging in to the Instance, where you do not yet have a User object, but still
+    /// need to make a RateLimited request.
+    pub(crate) fn shell(instance: Rc<RefCell<Instance>>, token: String) -> UserMeta {
+        let settings = UserSettings::default();
+        let object = User::default();
+        UserMeta {
+            belongs_to: instance.clone(),
+            token,
+            limits: instance.borrow().limits.clone(),
+            settings,
+            object,
+        }
+    }
```

Comment:
`User::get()` has to be public, and this also makes sense, because someone using this library might want to get information about another user or the user currently logged in. `shell` is a workaround for when you already have a token, but no other information about the user, and still need to make a rate-limited request to the server. This is only the case when you are performing a registration or login. This is why `shell` is not public: It should not have to be used by anyone using the library, unlike `User::get()`